andela/wolfsbane-backend
# #167750039 Setup Sequelize ORM with PostgreSQL

#### Description of Task to be completed?
- Setup Sequelize ORM with PostgreSQL
- Create models and migrations structure
- Create a Dummy User table
 #### How should this be manually tested?
- clone the repo
- cd into the root folder of the cloned repo
- run `npm install`  to install necessary dependencies
- run `npm install -g sequelize-cli` 
- open the terminal and type `touch .env` in the terminal
- ensure you have Postgres installed and setup
- create your database type  `createdb <db-name>`
- open the .env file and put your database info inside, check out the .env-example file in the root of the repo to see how to do this
- run `npm run migrate` to create your tables
 #### Any background context you want to provide?
 #### What are the relevant pivotal tracker stories?
[#167750044](https://www.pivotaltracker.com/story/show/167750044), [#167750039](https://www.pivotaltracker.com/story/show/167750039)
 #### Screenshots (if appropriate)
None
 #### Questions:
None
